ql/ql/src/queries/style/NoUppercaseVariables.ql

```diff
@@ -0,0 +1,42 @@
+/**
+ * @name No upper case variables
+ * @description Variables/fields/predicates should be lower-case, classes/modules should be upper-case
+ * @kind problem
+ * @problem.severity error
+ * @id ql/no-upper-case-variables
```

Comment:
The query id, name and file name suggest that this query is only about lower-case variables, but it's evolved to be about all declarations.